packages/canary-web-components/src/components/ic-date-input/ic-date-input.tsx

```diff
@@ -351,9 +350,11 @@ export class DateInput {
   }
 
   componentWillUpdate(): void {
-    if (!this.isDateSetFromKeyboardEvent) {
-      this.setDate(this.value);
-    }
+    if (isEmptyString(this.helperText))
```

Comment:
With this update, what happens if `null` is passed as helperText?

Reply:
I've added a null check too :)